cmd/main.go

```diff
@@ -249,6 +233,43 @@ func main() {
 		StartNSXServiceAccountController(mgr, commonService)
 	}
 
+}
+
+func waitForEntry(mgr manager.Manager, nsxClient *nsx.Client) {
+	log.Info("I'm trying to transition master")
+	<-mgr.Elected()
```

Comment:
"I'm trying to transition master"-> Electing master ?

Reply:
https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/manager/manager_test.go#L200
do you think we need to consider election canceled case like above?

Reply:
I'm trying to be elected as master